src/fmu/tools/rms/generate_bw_per_facies.py

```diff
@@ -0,0 +1,102 @@
+import copy
+from typing import Dict, List
+
+import numpy as np
+import xtgeo
+
+
+def create_bw_per_facies(
+    project,
+    grid_name: str,
+    bw_name: str,
+    original_petro_log_names: List[str],
+    facies_log_name: str,
+    facies_code_names: Dict[int, str],
+    debug_print: bool = False,
+) -> None:
+    """Script to be used as RMS python job to create new petrophysical
+    logs from original logs, but with one log per facies.
+    All grid blocks for the blocked wells not belonging to the facies
+    is set to undefined.
+
+    Purpose
+
+        Create the blocked well logs to be used to condition
+        petrophysical realizations where all grid cells are
+        assumed to belong to only one facies. This script will
+        not modify any of the original logs, only create new logs where only
+        petrophysical log values for one facies is selected
+        and all other are set ot undefined.
+
+    Input
+
+        grid model name,
```

Comment:
It would be better to use standard "google" docstring style here, see example here:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I tried.